lib/msal-angular/package.json

```diff
@@ -66,7 +66,8 @@
     "ts-node": "~8.3.0",
     "tslib": "^2.0.0",
     "typescript": "~4.1.5",
-    "zone.js": "~0.10.3"
+    "zone.js": "~0.10.3",
+    "engine.io": ">=6.2.1"
```

Comment:
just curious, isn't this a nested dependency? Shouldn't we use [overrides](https://docs.npmjs.com/cli/v9/configuring-npm/package-json#overrides) for this?